packages/api/src/libs/utils.test.ts

```diff
@@ -60,11 +63,19 @@ describe("utils", function () {
   });
   it("calculateTvl", function () {
     const emp = {
-      totalPositionCollateral: "1000000000",
+      totalPositionCollateral: parseUnits("100", 8).toString(),
```

Comment:
tip: can also grab `parseFixed` from `@uma/common` which is based on another implementation from the [`ethers` project](https://github.com/ethers-io/ethers.js/tree/master/packages/bignumber)